src/gridstack.ts

```diff
@@ -2306,7 +2260,7 @@ export class GridStack {
           this.resizeToContentCheck(false, node);
           if (subGrid) {
             subGrid.parentGridNode = node;
-            if (!subGrid.opts.styleInHead) subGrid._updateStyles(true); // re-create sub-grid styles now that we've moved
+            subGrid._updateStyles(); // re-create sub-grid styles now that we've moved

```

Comment:
might not be needed. all _updateStyles() need to be verified if needed to be called actually... sicne the external file isn't used now.

Reply:
Well, my changes are quite conservative, I don't have the full knowledge about the side effects expected by this call (i.e. _updateContainerHeight)

Reply:
fair enough. but if there was a condition styleInHead then that means it's location based and therefore entire thing doesn't apply.